### ISSUE TITLE
Large numbers become invalid strings after formatting

### ISSUE THREAD

**Issue by milieu** [awslabs/aws-cloudformation-template-formatter#10]
```
          -         autovacuum_freeze_max_age: 500000000
          +         autovacuum_freeze_max_age: 5e+08
```
The formatter introduced this diff and caused this Amazon Cloudformation error: `DBParameterGroup invalid parameter value: 5.0E8`

**Comment by stilvoid** [comment]
Thanks for raising this. Looks like the json unmarshaler is using an innapropriate type for numbers. I'll look into this today.

**Comment by stilvoid** [comment]
Fixed in v1.1.1
